docs/platforms/javascript/common/deployment-provider-setup/netlify.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Netlify
+sidebar_order: 10
+description: "Learn about running Sentry when deploying to Netlify."
+supported:
+  - javascript.nuxt
+---
+
+<Alert level="warning">
+  Deploying on Netlify is still experimental. Currently, not all Sentry features are supported on the server-side. Only your frontend will be monitored.
+</Alert>
+
+<Alert level="info">
+  Before approaching the steps below, read the general documentation around <PlatformLink to="/deployment-provider-setup/general/">setting up Sentry on a deployment provider</PlatformLink> to get quick general overview of the process.
+  This page only shows Netlify-specific steps.
+</Alert>
+
+## Approach 1: Adding the `--import` flag
+
+<PlatformSection supported={['javascript.nuxt']}>
+  The server-side of Nuxt is deployed as a Netlify Function. So make sure to scope the environment variable `NODE_OPTIONS` to "Functions".
+  The path to the function is depending on your setup. When building the preset locally, the file is available at: `.netlify/functions-internal/server/sentry.server.config.mjs`.
+
+  <Alert level="warning">
+    The docs for this are still vague and we will update them as soon as we know more about which exact path to use for the `--import` flag.
+  </Alert>
+
+  <Include name="common-imgs/netlify-env-variable" />
+</PlatformSection>
+
+
+## Approach 2: Enable `experimental_basicServerTracing`
+
+
+Enabling `experimental_basicServerTracing` can be used when adding the node option `--import` does not work. This however only comes with limited tracing functionality because Sentry server config is not **pre**loaded with `--import`.
+
+<PlatformSection supported={['javascript.nuxt']}>
+  Enabling this option will automatically import the Sentry server config at the top of the server entry file (server entry: `.netlify/functions-internal/server/server.mjs`).
+
```

Comment:
```suggestion

If adding the node option `--import` doesn't work, you can enable `experimental_basicServerTracing` instead. Note, that `experimental_basicServerTracing` only comes with limited tracing functionality because Sentry server config is not **pre**loaded with `--import`.

<PlatformSection supported={['javascript.nuxt']}>
  Enabling this option will automatically import the Sentry server config at the top of the server entry file (server entry: `.netlify/functions-internal/server/server.mjs`).

```